ui/apps/ui/src/app/dictionary/runDictionaryForInteroperabilityGuidelines.ts

```diff
@@ -0,0 +1,22 @@
+import { interoperabilityGuidelinesStatusDictionary } from '../dictionary/interoperabilityGuidelinesStatusDictionary';
+import { interoperabilityGuidelinesTypeDictionary } from '../dictionary/interoperabilityGuidelinesTypeDictionary';
+import { interoperabilityGuidelinesDomainDictionary } from '../dictionary/interoperabilityGuidelinesDomainDictionary';
+
+export function runDictionaryForInteroperabilityGuidelines(
```

Comment:
I would suggest some more concise function name. Also - return type is not defined

Reply:
refactor after merge